docs/rules/require-default-props.md

```diff
@@ -184,6 +184,91 @@ NotAComponent.propTypes = {
 };
 ```
 
+## Rule Options
+
+```js
+...
+"react/require-default-props": [<enabled>, { forbidDefaultForRequired: <boolean> }]
+...
+```
+
+* `enabled`: for enabling the rule. 0=off, 1=warn, 2=error. Defaults to 0.
```

Comment:
This is basic eslint documentation; individual rule docs don't tend to explain this.

Reply:
I just modified this block from another rule. Should I remove it?

Reply:
Not a blocker then, altho i don’t think any rules should have this.